packages/wonder-blocks-button/components/button.js

```diff
@@ -27,13 +34,6 @@ export type SharedProps = {|
      */
     spinner: boolean,
 
-    /**
-     * This should be use when `spinner={true}` to let people using screen
-     * readers that the action taken by clicking the button will take some
-     * time to complete.
-     */
-    "aria-label": string,
```

Comment:
I notice there is a `aria-label` defined in the `defaultProps`, we should probably remove that as well.